src/binaries/BinariesManager.ts

```diff
@@ -19,25 +18,13 @@ export class BinariesManager {
                                      accountContextManager: AccountContextManager,
                                      expectedCLIVersion: string
     ): IBinariesUtility {
-        if (this.useBinariesUtilityV2()) {
-            logger.trace(TelemetryEvent.BinariesUtility_VersionV2);
+        if (this.useBinariesUtility()) {
+            logger.trace(TelemetryEvent.BinariesUtility_Version);
             return new BinariesUtilityV2(logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
```

Comment:
Should we also rename the BinariesUtilityV2 class now that V1 isn't a scenario anymore?

Reply:
Hey Dan! since this is my first PR, I made minimal changes to keep thing simple. Yes, going forward we will be renaming all the variable and classes with V2 to indicate only one version.